internal/action/repl.go

```diff
@@ -13,6 +13,31 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+type mindFlagsCompleter struct {
+	base readline.AutoCompleter
+}
+
+func (f *mindFlagsCompleter) Do(line []rune, pos int) ([][]rune, int) {
+	// Get the portion of the line up to the cursor and split to tokens
+	origInput := string(line[:pos])
+	tokens := strings.Fields(origInput)
+	var filtered []string
+	for _, tok := range tokens {
+		// Skip flag tokens
+		if strings.HasPrefix(tok, "-") {
+			continue
+		}
+		filtered = append(filtered, tok)
+	}
+	sanitized := strings.Join(filtered, " ")
+	// Add additional space if had one on original line
+	if len(origInput) > 0 && origInput[len(origInput)-1] == ' ' {
+		sanitized += " "
+	}
+	return f.base.Do([]rune(sanitized), len([]rune(sanitized)))
```

Comment:
We require a blank link before the return statement to make it stand out better.